### PR TITLE
adding readme on start coding

### PR DESCRIPTION
**Pull request by morsh**
I've added a readme on how to start coding.

### CODE REVIEW

**Comment by jkleinsc** [comment]
Thanks @morsh.  @HospitalRun/core-maintainers what do you think of this PR?

**Comment by jglovier** [comment]
This is fantastic. :clap: :sparkles:

Haven't had a chance to really dig into the content yet, but only feedback at a glance is where it should live. I think this should probably be included as a section in the CONTRIBUTING.md file.

**Comment by morsh** [comment]
Hi @jglovier 
I'm guessing this is obsolete by now?

**Comment by jkleinsc** [comment]
@morsh I think the information is still relevant, but it needs to be placed here https://github.com/hospitalrun/hospitalrun-frontend/blob/master/.github/CONTRIBUTING.md instead of in its own file

**Comment by morsh** [comment]
Hi @jkleinsc,
I moved the changes to CONTRIBUTING.md under a new section and updated the references accordingly.

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -33,3 +33,108 @@ This guide assumes a level of knowledge on how to use GitHub issues and pull req
 - Please keep your changes succinct in scope. Large pull requests with multiple changes to the codebase (i.e. multiple features) may be asked to be broken into multiple smaller pull requests.
 - Please give as much detail as is relevant about your approach to the solution in the pull request description.
 - When your code changes include any change to the visual UI, please include a screenshot of how your changes appear in the browser.
+
+## Start Coding
+
+Thissection is designed to help developers start coding in the project and understanding the basic concepts and components used.
```
Comment (jkleinsc) [inline .github/CONTRIBUTING.md]: Needs space between "This" and "section"

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -33,3 +33,108 @@ This guide assumes a level of knowledge on how to use GitHub issues and pull req
...
+
+### Ember
+
+To understand the project you'll have to understand Ember, and it is advisable that you'll follow the tutorial: [Create your own app](https://guides.emberjs.com/v2.6.0/tutorial/ember-cli/) to get an understanding of how ember works and the basic folder structure.
```
Comment (jkleinsc) [inline .github/CONTRIBUTING.md]: The tutorial guide url should be updated to:
https://guides.emberjs.com/v2.9.0/tutorial/ember-cli/

**Comment by morsh** [comment]
@jkleinsc - done

**Comment by jkleinsc** [comment]
Thanks @morsh!  I'll merge it in.

**Pull request merged**